alacritty/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 [dependencies]
 alacritty_terminal = { path = "../alacritty_terminal", default-features = false }
 clap = "2"
-log = "0.4"
+log = { version = "0.4", features = ["std"] }
```

Comment:
This was necessary for builds with `--no-default-features` to work.

Reply:
That seems strange to me? `--no-default-features` should only affect the root crate, not its dependencies, right?

On the latest master it seems to work just fine?

Reply:
Yeah hmm, it does indeed work on latest master with `--no-default-features`. On this branch, without this change, I get:

```
> cargo run --no-default-features
   Compiling alacritty v0.5.0-dev (C:\Users\david\dev\alacritty\alacritty)
error[E0425]: cannot find function `set_boxed_logger` in crate `log`
  --> alacritty\src\logging.rs:47:10
   |
47 |     log::set_boxed_logger(Box::new(logger))?;
   |          ^^^^^^^^^^^^^^^^ not found in `log`
```

Reply:
Which directory are you running that command from?

Reply:
The `alacritty` subdirectory.

Reply:
I've just checked and the `log` crate does not have any default features (see https://github.com/rust-lang/log/blob/master/Cargo.toml). Could you confirm again this actually prevents building?

Reply:
Hmm yes, it's definitely still an issue for me on this branch. Technically it's not wrong, as docs for `set_boxed_logger` state that `std` feature needs to be present, but it's weird that it's not an issue on master...

Reply:
Hmm, this might be due to a different dependency opting into that feature. Cargo's a bit special about that. Should be good, thanks.